charts/castai-db-optimizer/templates/envoy_config.yaml

```diff
@@ -20,12 +20,12 @@ data:
           path: /data/cache
           db_size_bytes: {{ .Values.resources.proxy.cacheDbSizeBytes | int }}
           max_cache_entries: {{ .Values.resources.proxy.maxCacheEntries | int }}
-          eviction_threshold: 0.5
-          stop_caching_threshold: 0.95
-          write_batch_size: 100
-          write_batch_delay_ms: 100
-          stats_thread_period_ms: 1000
-          eviction_thread_period_ms: 1000
+          eviction_threshold: {{ .Values.proxy.evictionThreshold }}}
```

Comment:
Let's ensure those statements have default values equal to the old ones so the generated output won't change for existing deployments. For example: 
`{{ .Values.proxy.evictionThreshold | default 0.5 }}`